File: Data Structures & Algorithms/concatenation-of-array/submission-2.java
class Solution {
    public int[] getConcatenation(int[] nums) {
        int len = nums.length;
        int[] results = new int[2 * len];
        for(int i=0; i<len; i++){;
            results[i] = nums[i];
            results[i+len] = nums[i];
        }
        return results;
    }
}

File: Data Structures & Algorithms/duplicate-integer/submission-2.js
class Solution {
    /**
     * @param {number[]} nums
     * @return {boolean}
     */
    hasDuplicate(nums) {
        nums.sort((a,b) => a-b)
        let prev = null
        for(let i of nums){
            if(i === prev) return true
            prev = i
        }
        return false
    }
}
